test/int.jl

```diff
@@ -176,6 +176,19 @@ end
 @test widemul(false, 3) == 0
 @test widemul(3, true) == widemul(true, 3) == 3
 
+let i=Int64(2)^63-1, k=widemul(i,i)
+
+    @test widemul(i,i)==85070591730234615847396907784232501249
+    
+    local j=div(k,2)
```

Comment:
`local` isn't necessary here. the `check-whitespace` failure is due to the indent on the empty next line